src/main/groovy/com/palantir/gradle/docker/DockerExtension.groovy

```diff
@@ -90,12 +93,12 @@ class DockerExtension {
     }
 
     public Set<String> getTags() {
-        return tags
+        return tags.get()
     }
 
     @Deprecated
     public void tags(String... args) {
-        this.tags = ImmutableSet.copyOf(args)
+        this.tags.set(ImmutableSet.copyOf(args))
```

Comment:
Hmm, from memory wasn't the way this used to work that people would put

```gradle
docker {
    tags 'latest'
}
```

but it would still make the version tag? I wonder if we just want to make `getTags()` return `tags.get() + [version]`?

Reply:
Updated to add the version tag to the other tags if it does not exist already.

However, I'm actually not 100% sure where the version tag came from before. E.g. a bunch of projects would not specify any tags and still get the correct publish.

Reply:
@CRogers, this was a bad change.  Now there's no way NOT to build the version tag (#516)